src/Service/Services/GraphQLSchemaCreator.cs

```diff
@@ -155,10 +156,10 @@ DatabaseType.postgresql or
                     IEnumerable<string> rolesAllowedForEntity = _authorizationResolver.GetRolesForEntity(entityName);
                     Dictionary<string, IEnumerable<string>> rolesAllowedForFields = new();
                     SourceDefinition sourceDefinition = _sqlMetadataProvider.GetSourceDefinition(entityName);
-                    bool isStoredProcedure = entity.ObjectType is SourceType.StoredProcedure;
+                    bool isStoredProcedure = entity.Source.Type is EntityType.StoredProcedure;
                     foreach (string column in sourceDefinition.Columns.Keys)
                     {
-                        Config.Operation operation = isStoredProcedure ? Config.Operation.Execute : Config.Operation.Read;
+                        Config.EntityActionOperation operation = isStoredProcedure ? Config.EntityActionOperation.Execute : Config.EntityActionOperation.Read;
```

Comment:
EntityActionOperation seems too verbose/redundant. I'm leaning towards EntityAction to match the "actions" section of the runtime config entity permissions. 

On another note, would renaming like this be better suited for a follow-PR to reduce the number of changes in this PR?

Reply:
I'm personally a fan of verbosity in types and variables to make it clear as to where a type relates to. In this case the Operation is off the EntityAction, which is why I named it as such.

In the previous version where it was called Operation there was some conflicts in type name being another type from the CLR.